src/highlight.js

```diff
@@ -593,6 +593,7 @@ https://highlightjs.org/
           } else if (options.tabReplace) {
             return p1.replace(/\t/g, options.tabReplace);
           }
+          return "";
```

Comment:
A tiny nit: could you use single quotes for strings as in the rest of the code? :-)